src/azul/transformer.py

```diff
@@ -300,6 +300,23 @@ def get(self):
         return thaw(super().get())
 
 
+class FrequencySetAccumulator(SetAccumulator):
+
+    def __init__(self, max_size=None) -> None:
+        super().__init__(max_size)
+        self.value = Counter()
+
+    def accumulate(self, value) -> bool:
+        if isinstance(value, (list, set)):
+            self.value.update(value)
+        else:
+            self.value.update([value])
```

Comment:
```suggestion
            self.value[value] += 1
```

… is shorter and more efficient.